Kinvey/Kinvey/Client.swift

```diff
@@ -256,6 +279,30 @@ open class Client: Credential {
         }
     }
     
+    /// Initialize a `Client` instance with all the needed parameters.
+    open func initialize<U: User>(
+        appKey: String,
+        appSecret: String,
+        instanceId: String,
+        accessGroup: String? = nil,
+        encryptionKey: Data? = nil,
+        schema: Schema? = nil,
+        options: Options? = nil,
+        completionHandler: @escaping (Result<U?, Swift.Error>) -> Void
+    ) {
+        return initialize(
```

Comment:
We should validate the `instanceId` parameter before using it? You are force-unwrapping the `URL()`, so if there are any invalid characters, the method will exception out. Maybe it's better to detect that and throw our own error instead.

Reply:
agree. done